action.yaml

```diff
@@ -22,6 +22,9 @@ inputs:
 runs:
   using: "composite"
   steps:
+    - name: Run update and upgrade
+      run: sudo apt update && sudo apt -y upgrade
```

Comment:
In theory that should be fine to force an apt upgrade since [the only linux-based runner proposed by github action is Ubuntu](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#choosing-github-hosted-runners). However, we don't have guarantee that users won't have [self-hosted runners](https://docs.github.com/en/actions/hosting-your-own-runners) with their distro of choice. Can we make this step more generic, or at least optional?

Reply:
Meh, forget what I said, there is already many other place where we assume ubuntu. If we were to allow other distros, this would require more work outside the scope of this PR.